oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/indexstore/IndexStoreUtils.java

```diff
@@ -39,6 +40,23 @@
 public class IndexStoreUtils {
     public static final String METADATA_SUFFIX = ".metadata";
 
+    public static final String OAK_INDEXER_USE_ZIP = "oak.indexer.useZip";
+    public static final String OAK_INDEXER_USE_LZ4 = "oak.indexer.useLZ4";
+
+    public static boolean compressionEnabled() {
```

Comment:
I think this is slightly ambiguous. It seems like USE_ZIP has two meanings: whether to enable compression **and** use GZIP to compress? But LZ4 only indicates whether we use LZ4. So if we disable ZIP but enable LZ4, it wouldn't be used. On the other hand, if we enable ZIP, we will use compression *and* GZIP.